tests/test_functional.py

```diff
@@ -51,19 +74,19 @@ def test_no_404s(pcic_data_portal, url):
 @pytest.mark.crmpdb
 @pytest.mark.parametrize(('url', 'title', 'body_strings'), [
     ('/data/pcds/lister/', 'PCDS Data',
-     ["Climatological calculations", "raw/"]),
+     [b"Climatological calculations", b"raw/"]),
     ('/data/pcds/lister/raw/', "Participating CRMP Networks",
-     ["FLNRO-WMB/", "Environment Canada (Canadian Daily Climate Data 2007)"]),
+     [b"FLNRO-WMB/", b"Environment Canada (Canadian Daily Climate Data 2007)"]),
     ('/data/pcds/lister/raw/AGRI/',
-     "Stations for network AGRI", ["de107/", "Deep Creek"]),
+     "Stations for network AGRI", [b"de107/", b"Deep Creek"]),
 ])
 def test_climo_index(
         pcic_data_portal, url, title, body_strings):
     req = Request.blank(url)
     resp = req.get_response(pcic_data_portal)
     assert resp.status == '200 OK'
     assert resp.content_type == 'text/html'
-    assert resp.content_length < 0
+    #assert resp.content_length < 0
```

Comment:
what's the deal with the previous version of this line? why would content length have been _less_ than zero at any point? Is this leftover from an intensive debugging process (possibly before you took over)?

Reply:
I have no idea what this is. I can get rid of it.